docs/central-install-digital-ocean.rst

```diff
@@ -565,6 +565,104 @@ DKIM is a security trust protocol which is used to help verify mail server ident
+
+Central users will be able to change their display name shown in Central and to choose a different name from what is shown in the identity provider. However, because Central identifies users by their email address, most users will not be allowed to change their email address. Only a Central Administrator will be able to change the email address associated with a Central account. That will be needed if a user's email address changes within the identity provider. In that case, an Administrator will need to manually change the user's email address within Central to match their new address in the identity provider.
+
+If a Central Administrator changes their own email address to one that does not match the identity provider, they may lose access to Central. If they are the only Administrator, they will need to disable SSO, set a password :ref:`by command line <central-command-line-user-set-password>`, log in, change their email address, then re-enable SSO.
+
+Logout is not centralized, which means that when a user logs out of Central, that will not log them out of the identity provider. Conversely, when a user logs out of the identity provider, that will not log them out of Central. If a user logs out of Central, then goes to log back in, they may find that login is nearly instantaneous if they are still logged into the identity provider. That is, they may find that they are not required to log into the identity provider again in order to log into Central.
+
+Enabling SSO in an Existing Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to enable SSO for an existing Central installation, even if the installation has existing users. Follow the instructions above to enable SSO. Because Central identifies users by their email address, the email address associated with a Central account must match the address in the identity provider. If the email address does not match, the user will not be able to log in.
+
+Enabling SSO will not log out users who are already logged in. Users who are already logged in will be required to log in through the identity provider only after being logged out.
+
+Disabling SSO
+~~~~~~~~~~~~~
+
+It is possible to disable SSO by following the steps below. If there were users before SSO was enabled (if SSO was disabled, then enabled, then disabled again), users will be able to log into Central using their same password from before SSO was enabled. You can :ref:`reset users' passwords <central-users-web-reset-password>` after disabling SSO.
+
+To disable SSO:
+
+#. In ``.env``, set ``OIDC_ENABLED`` to ``false``. Clear ``OIDC_ISSUER_URL``, ``OIDC_CLIENT_ID``, and ``OIDC_CLIENT_SECRET``.
```

Comment:
It's not strictly necessary to clear the latter three variables. We probably won't clear them on our dev servers. Our dev servers are unusual in that we will often toggle SSO on and off. However, that won't be the case for most users. I think it's easiest just to tell users to clear these variables rather than that they may clear them, but aren't required to. If they don't need to keep these values on the server, it's probably best to clear them, as some of them are sensitive.